scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -35,7 +35,7 @@ function create_subm_vars() {
     subm_colorcodes=blue
     subm_debug=false
     subm_broker=k8s
-    subm_cabledriver=strongswan
+    subm_cabledriver=libreswan
```

Comment:
So we will default to libreswan?

I think if we want to make this change it should be on the operator itself, when we do it.

Reply:
https://github.com/submariner-io/shipyard/pull/365 will set libreswan as the default. You're saying we should set it to strongswan there?

Reply:
The _default_ default (_i.e._, the cable driver used when Submariner isn’t told which to use) is determined by Submariner itself, see https://github.com/submariner-io/submariner/pull/972 which changes it to Libreswan. The operator can have a different default, as can `subctl` and the various test suites ;-). But it’s better IMO if we align everything on Libreswan for 0.8.